warehouse/ingest-configuration/src/main/resources/config/shard-ingest-config.xml

```diff
@@ -89,4 +89,8 @@
     <name>partitioner.category.member.${table.name.shard}</name>
     <value>shardedTables</value>
 </property>
+<property>
+    <name>shard.keep.count.only.index.entries</name>
```

Comment:
Elizabeth requested a rename here to `index.tables.keep.count.only.entries`. I'll make that update.